src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -145,7 +149,7 @@ function NavItem({ hostURLConfig, serverMode }) {
             component: HostURLConfiguration,
             isAvailableInEA: false,
         },
-        { name: 'GitOps ', href: URLS.GLOBAL_CONFIG_GITOPS, component: GitOpsConfiguration, isAvailableInEA: false },
```

Comment:
Is removal of `isAvailableInEA` expected?

Reply:
yes